### PR TITLE
Basic db support for development purposes

### PR DESCRIPTION
**Pull request by ivange94**
This PR adds support for sqlite db. This will do for development purposes.

### CODE REVIEW

File: pkg/models/db.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
```
Comment (acha-bill) [inline pkg/models/db.go]: Encapsulate this field. 
You can use `sync.Once` to get a singleton.

File: pkg/handlers/user_handlers.go [hunk 2]
```diff
@@ -12,15 +12,13 @@ func CreateUser(c echo.Context) error {
...
-
-	u := &models.User{
-		Name:     "John Doe",
-		Email:    "john@example.com",
-		Username: c.Param("username"),
-	}
-	return c.JSON(http.StatusOK, u)
+	username := c.QueryParam("username")
+	var user models.User
+	models.DB.First(&user, "username = ?", username)
```
Comment (acha-bill) [inline pkg/handlers/user_handlers.go]: It'll make sense for this to be in `models` pkg. Consider a `pkg/models/user_model` file where you manage all user/db interactions. 

That way, `models.DB` obj is local to `models` pkg.

**Review by acha-bill — CHANGES_REQUESTED** [review]
Did you commit `test.db` intentionally?

**Comment by ivange94** [comment]
> Did you commit `test.db` intentionally?

That was a mistake. Will fix 😅

**Pull request merged**